Makefile

```diff
@@ -1,10 +1,49 @@
-deps-update:
+# Go parameters
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+GOCMD=go
+GOBUILD=$(GOCMD) build --mod=vendor
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+BINARY_NAME=./bin/bond
+
+# the following vars are overidable with ENV vars
+# example:
+#  before running make set vars like so
+#  export DOCKER_REPO=my_repo
+IMAGE_REPO?=jdambly
```

Comment:
These aren't good defaults - it's better to leave this with very generic default values for the repo etc. unless we have a stable known host for the image.
I think we want it to be clear that the image is not available from a public registry

Reply:
sure I can remove this